modules/artic.nf

```diff
@@ -134,6 +135,8 @@ process articMinIONNanopolish {
     --sequencing-summary ${seqSummary} \
     ${params.scheme}/${params.schemeVersion} \
     ${sampleName}
+
+    gunzip ${sampleName}.pass.vcf.gz
```

Comment:
Believe VCF module will gunzip for you if you pass the .gz file. Code of [reader](https://pyvcf.readthedocs.io/en/v0.3.0/_modules/vcf.html)

Reply:
Changed, good call. Testing it now